.gitignore

```diff
@@ -14,3 +14,9 @@ local.properties
 /.settings/*
 .idea/
 *.iml
+
+# Gererated file
+ .DS_Store
+ .DS_Store?
+ desktop.ini
+ Thumbs.db
```

Comment:
Please do not commit this here. I have prepared a much more complete MR in #501 for this (which would conflict with this one).